inject-groovy/src/test/groovy/io/micronaut/inject/field/FieldInjectionSpec.groovy

```diff
@@ -166,5 +178,31 @@ class FieldInjectionSpec extends Specification {
         }
     }
 
+    static class F {
```

Comment:
Is this class used somewhere else? Doesn't seem to be used in this spec

Reply:
Good catch, I have corrected the test